crates/punktf-cli/src/util.rs

```diff
@@ -148,16 +287,28 @@ pub fn log_deployment(deployment: &Deployment) {
 		}
 	};
 
-	let files_total = files_success + files_skipped + files_failed;
+	let dotfiles_total = dotfiles_success + dotfiles_skipped + dotfiles_failed;
+	let links_total = links_success + links_skipped + links_failed;
+
 	let elapsed = deployment
 		.duration()
 		.expect("Failed to get duration from deployment");
 
 	write!(out, "\nTime            : {:?}", elapsed).expect("Write to String failed");
-	write!(out, "\nFiles (deployed): {}", files_success).expect("Write to String failed");
-	write!(out, "\nFiles (skipped) : {}", files_skipped).expect("Write to String failed");
-	write!(out, "\nFiles (failed)  : {}", files_failed).expect("Write to String failed");
-	write!(out, "\nFiles (total)   : {}", files_total).expect("Write to String failed");
+	write!(out, "\n{}", "-".repeat(80).dimmed()).expect("Write to String failed");
+	write!(out, "\nFiles (deployed): {}", dotfiles_success).expect("Write to String failed");
+	write!(out, "\nFiles (skipped) : {}", dotfiles_skipped).expect("Write to String failed");
+	write!(out, "\nFiles (failed)  : {}", dotfiles_failed).expect("Write to String failed");
+	write!(out, "\nFiles (total)   : {}", dotfiles_total).expect("Write to String failed");
+	write!(out, "\n{}", "-".repeat(80).dimmed()).expect("Write to String failed");
+	write!(out, "\nLinks (deployed): {}", links_success).expect("Write to String failed");
+	write!(out, "\nLinks (skipped) : {}", links_skipped).expect("Write to String failed");
+	write!(out, "\nLinks (failed)  : {}", links_failed).expect("Write to String failed");
+	write!(out, "\nLinks (total)   : {}", links_total).expect("Write to String failed");
 
-	log::info!("{}", out);
+	if print {
```

Comment:
i saw this before, put it in a utility fn

Reply:
Is now. Thanks!